Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -852,6 +854,13 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'Era' : 'Phase2C12',
         'ScenToRun' : ['GenSimHLBeamSpotFull','DigiFullTrigger','RecoFullGlobal', 'HARVESTFullGlobal'],
     },
+    '2026D59' : {
+        'Geom' : 'Extended2026D59',
+        'HLTmenu': '@fake2',
+        'GT' : 'auto:phase2_realistic_T15',
+        'Era' : 'Phase2C11',
```

Comment:
Isn't it misleading to couple `Phase2C11` with `C13` geometry?

Reply:
We have only defined Phase2C11 which will work for V11, V12, V13 geometries of HGCal like Phase2C9 which works for V9 and V10